source/js/main-page-controller.js

```diff
@@ -65,62 +69,78 @@ export class MainPageController {
     }
 
     // Toolbar buttons
-    let textTool = document
-      .querySelector('tool-bar')
-      .shadowRoot.querySelector('#text-tool')
+    let textTool = this.toolbar.shadowRoot.querySelector('#text-tool')
 
     textTool.addEventListener('click', (event) => {
```

Comment:
Seems like when user clicks on TextTool, it remains highlighted when they click on it. Makes it confusing for the user since they do not know if the TextTool is on or off.

Reply:
The button or the textbox?

Reply:
The button. When you click on it, you know how the button background gets darker letting the user know they selected that tool. But let's say they want to turn off that tool. You would click on it again and it would turn off but the background remains dark of the Toolbar so the user doesn't really know if it's on or off.

Reply:
Oh I see, the button remains selected because as long as the button is highlighted, the user can insert a new textbox as well, once they select some other tool, textbox is no longer highlighted, and they cannot add a new textbox. 
We could however make changes so that upon clunselecting the tool

Reply:
Yes, great idea. We can switch its css class when the mousedown event happen and change it back again when mouseup. I create a new issue for it.